problem1.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+class Solution {
+
+    static int solveMeFirst(int a, int b) { 
+      // Hint: Type return a+b; below  
+      int c=a+b;
```

Comment:
No need to use extra variable "c". You can directly return a+b. One extra line of code means extra maintenance. Please keep this in mind

Reply:
changed name and done all the changes in code.